sirepo/nersc.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""Run Python processes in background
```

Comment:
Copy and paste

Reply:
[edebf8a](https://github.com/radiasoft/sirepo/pull/5493/commits/edebf8ac2b64f6de408bcb80d56d5324c8b57181)